doc/man/scons.xml

```diff
@@ -1120,24 +1120,48 @@ the mechanisms in the specified order.</para>
   </listitem>
   </varlistentry>
 
+  <varlistentry id="opt--experimental">
+    <term>
+      <option>--experimental=<replaceable>all</replaceable></option>,
+      <option>--experimental=<replaceable>none</replaceable></option>
```

Comment:
I guess `none` is useful for temporary override if someone used `SetOption` to enable an experimental?

Reply:
Exactly!